projects/fal/src/fal/api.py

```diff
@@ -1101,6 +1103,17 @@ async def app_exception_handler(request: Request, exc: AppException):
         async def field_exception_handler(request: Request, exc: FieldException):
             return JSONResponse(exc.to_pydantic_format(), exc.status_code)
 
+        # ref: https://github.com/fastapi/fastapi/blob/37c8e7d76b4b47eb2c4cced6b4de59eb3d5f08eb/fastapi/exception_handlers.py#L20
+        @_app.exception_handler(RequestValidationError)
+        async def request_val_exception_handler(
+            request: Request, exc: RequestValidationError
+        ):
+            return JSONResponse(
+                {"detail": jsonable_encoder(exc.errors())},
+                422,
+                headers={"x-fal-billable-units": "0"},
```

Comment:
I am not entirely sure this applies to all. our billing can vary depending on this header being or not there

Reply:
also some 422s should be charged, as they incur in cost

Reply:
to be discussed internally